app/controllers/auth/LocalPageVisibilityPredicate.scala

```diff
@@ -74,9 +73,10 @@ trait LocalConfidenceLevelPredicate extends PageVisibilityPredicate with Confide
     if ( userHasHighConfidenceLevel(authContext) )
       Future.successful(PageIsVisible)
     else {
-      allowLowConfidenceSAEnabled match {
-        case true => Future.successful(new PageBlocked(Future.successful(Redirect(routes.ApplicationController.ivExemptLandingPage(successUrl)))))
-        case false => Future.successful(new PageBlocked(Future.successful(buildIVUpliftUrl(ConfidenceLevel.L200))))
+      if(allowLowConfidenceSAEnabled) {
```

Comment:
nice refactor! we can chain these if else if youd like to rather than nesting them. Up to you which you feel is more readable :+1:

Reply:
Chained those if-else statements